### PR TITLE
[TTAHUB-934] Update approved report template

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
- Add a "version" column to the AR, which will determine which approved AR form to display
- Create new component to match [the mockup](https://www.figma.com/file/rJGMygyqMz8jjCcoEpzcnK/Activity-Reports?node-id=3786%3A123568) for the redesigned AR. (It's "Approved Activity Report - no borders" at the top)

## How to test
- Run the migration, confirm that all existing reports have either 0 or 1 as their version number, depending on whether they were legacy reports.
- Check existing reports, ensure that the layout on the approved report page hasn't changed. Also verify print view still looks ok.
- Create a new report, approve it, and view the approved report template. Match it to the mockup. Check the print view, make sure that it looks ok. You can verify that it has the correct version.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-934

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/ActivityReport/index.js [hunk 1]
```diff
@@ -539,6 +540,7 @@ function ActivityReport({
             endDate: endDateToSave,
             regionId: formData.regionId,
             approverUserIds: approverIds,
+            version: 2,
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/ActivityReport/index.js:543]: Isn't this path for a new report? Why do we set version to 2?

Comment (thewatermethod) [reply frontend/src/pages/ActivityReport/index.js:543]: All reports saved after this change goes through will be version 2 reports

File: frontend/src/pages/ApprovedActivityReport/components/ApprovedReportSection.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+.ttahub-approved-report-section--heading--section-row > p {
+  flex-basis: max(300px, 75%)
+}
+
+.ttahub-approved-report-section--heading--section-row-title {
+  flex-basis: min(220px, 25%);
+}
+
+// a reset to squeeze those ul's in
+.ttahub-approved-report-section--heading--section-row > p > ul {
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/ApprovedActivityReport/components/ApprovedReportSection.scss:13]: Not sure if we can use the USWD pre defined classes here?

File: frontend/src/pages/ApprovedActivityReport/index.js [hunk 3]
```diff
@@ -1,144 +1,24 @@
+/* eslint-disable no-unused-vars */
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/ApprovedActivityReport/index.js:1]: Do we need this?

File: frontend/src/pages/ApprovedActivityReport/index.scss [hunk 4]
```diff
@@ -1,56 +1,63 @@
...
+    margin: 0;
+  }
     
-    /* how clumsy it is to set padding-top 0 to a class named padding 5? */
-    .ttahub-activity-report-view > .padding-5 {
-        padding: 0;
-    }
+  /* how clumsy it is to set padding-top 0 to a class named padding 5? */
+  .ttahub-activity-report-view > .padding-5 {
+    padding: 0;
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/ApprovedActivityReport/index.scss:61]: Should this be padding-top or padding? (based on what you say in the comment)

Comment (thewatermethod) [reply frontend/src/pages/ApprovedActivityReport/index.scss:61]: that's an old comment, I just cleaned up the spacing in this file, I am going to leave alone for now

**Review by AdamAdHocTeam — COMMENTED** [review]
Get this props validation viewing a V1 report.

image

File: frontend/src/pages/ActivityReport/index.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply frontend/src/pages/ActivityReport/index.js:543]: Yea that's my bad thought we were incrementing the value.

**Review by AdamAdHocTeam — COMMENTED** [review]
Do we want the border around activity and goal sections? Right now I don't see it on a V2.

image

**Review by AdamAdHocTeam — COMMENTED** [review]
The mockup has that we show "None provided" if there are no resource attachments.

**Review by AdamAdHocTeam — COMMENTED** [review]
Creator vs Manager notes should be different row colors per mockup.

image

**Comment by thewatermethod** [comment]
> Creator vs Manager notes should be different row colors per mockup.

In the mockup, that striping on that one section behaves differently then all the rest, so I propose we just let it ride for consistency's sake and not having to add a one-off case.

File: frontend/src/pages/ApprovedActivityReport/index.js (hunk 3 above)
Comment (thewatermethod) [reply frontend/src/pages/ApprovedActivityReport/index.js]: good catch, thanks, I removed

**Review by AdamAdHocTeam — COMMENTED** [review]
Can you double check our font size for the various levels. It seems like are headers and details have a larger size.

image

**Review by AdamAdHocTeam — COMMENTED** [review]
Can we fix this by adding a unique key on V2.

image

**Pull request merged**